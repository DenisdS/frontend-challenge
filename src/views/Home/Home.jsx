import React from 'react'

import { Link } from 'react-router-dom'

import { SubHeader } from 'components'
import { SomosClient } from 'utils'

import styles from './Home.module.css'
class Home extends React.Component {

  client = new SomosClient()

  render() {
    return (
      <React.Fragment>
        <SubHeader
          breadcrumb={[{ text: 'Home' }]}
          heading="Somos front-end Challenge"
        />
        <div className={styles.wrapper}>
          <h1>Escute seu artista preferido!</h1>
          <Link to="/busca">Buscar Artista</Link>
        </div>
      </React.Fragment>
    )
  }
}

export default Home
