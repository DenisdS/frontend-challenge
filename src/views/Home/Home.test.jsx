import React from 'react'

import { BrowserRouter } from 'react-router-dom'
import toJson from 'enzyme-to-json'

import { shallow } from '../../enzyme'

import Home from './Index'

import {
  SubHeader
} from 'components'


describe('Home - views', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <Home />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})

