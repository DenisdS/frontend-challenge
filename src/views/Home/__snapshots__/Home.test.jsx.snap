// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home - views should match snapshot 1`] = `
<Fragment>
  <SubHeader
    breadcrumb={
      Array [
        Object {
          "text": "Home",
        },
      ]
    }
    heading="Somos front-end Challenge"
  />
  <div
    className="wrapper"
  >
    <h1>
      Escute seu artista preferido!
    </h1>
    <Link
      replace={false}
      to="/busca"
    >
      Buscar Artista
    </Link>
  </div>
</Fragment>
`;
